packages/bitcore-client/src/storage.ts

```diff
@@ -41,10 +41,21 @@ export class Storage {
     }
   }
 
+  async verifyDbs(dbs) {
+    // test mongo connection
+    const mongoDbIdx = dbs.findIndex(d => d instanceof Mongo);
+    if (!(await (dbs[mongoDbIdx] as Mongo).testConnection())) {
```

Comment:
A cleaner way of doing this would be to loop over every val in dbs and either: 
- add a `testConnection` to the other storage type classes (they could be stubbed to always return true) 
OR
- do `if (typeof db.testConnection === 'function') {...`